lib/ansible/modules/system/ufw.py

```diff
@@ -416,6 +418,8 @@ def ufw_version():
                 execute(cmd + [[command], [value], [params['direction']]])
 
         elif command == 'rule':
+            if params['direction'] in ['outgoing', 'incoming', 'routed']:
```

Comment:
Same as above:
```suggestion
            if params['direction'] not in ['in', 'out']:
```

Reply:
That one needs a `None` as well:
```suggestion
            if params['direction'] not in ['in', 'out', None]:
```